From 1b22e4fd7741e1347f75c6bb062b9a26500cdb8a Mon Sep 17 00:00:00 2001
From: Christoph Gohlke <cgohlke@cgohlke.com>
Date: Sat, 12 Nov 2022 12:51:07 -0800
Subject: [PATCH] Fix build errors on Windows for ARM64

---
 hdf/src/hdfi.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hdf/src/hdfi.h b/hdf/src/hdfi.h
index a37c53f00..36bed742e 100644
--- a/hdf/src/hdfi.h
+++ b/hdf/src/hdfi.h
@@ -552,7 +552,7 @@ typedef long            hdf_pint_t;   /* an integer the same size as a pointer *
 /* Metrowerks Mac compiler defines some PC stuff so need to exclude this on the Mac */
 #if !(defined (__APPLE__)) && !(defined(__CYGWIN__))
 
-#if defined _M_ALPHA || defined _M_X64 || defined _M_IA64 || defined _M_IX86 || defined INTEL86 || defined M_I86 || defined M_I386 || defined DOS386 || defined __i386 || defined UNIX386 || defined i386
+#if defined _M_ARM64 || defined _M_ALPHA || defined _M_X64 || defined _M_IA64 || defined _M_IX86 || defined INTEL86 || defined M_I86 || defined M_I386 || defined DOS386 || defined __i386 || defined UNIX386 || defined i386
 #ifndef INTEL86
 #define INTEL86
 #endif /* INTEL86 */
